sharepa/analysis.py

```diff
@@ -16,6 +16,33 @@ def bucket_to_dataframe(name, bucket):
     ).T
 
 
+def convert_key(x):
+    x.key = x.key_as_string
+    return x
+
+
+def aggregation_to_dataframe(name,agg,filter=None):
```

Comment:
true that. Yep, good idea, ill give that a go

Regards,
Ben Yetton

Graduate Student,
Mednick Sleep and Cognition Lab,
University Of California, Riverside
900 University Ave
Riverside, CA 92521
benjamin.yetton@email.ucr.edu
On Jun 17, 2015 9:12 AM, "Fabian von Feilitzsch" <notifications@github.com>
wrote:

> In sharepa/analysis.py
> <https://github.com/fabianvf/sharepa/pull/3#discussion_r32624148>:
>
> > @@ -16,6 +16,33 @@ def bucket_to_dataframe(name, bucket):
> >      ).T
> >
> >
> > +def convert_key(x):
> > +    x.key = x.key_as_string
> > +    return x
> > +
> > +
> > +def aggregation_to_dataframe(name,agg,filter=None):
>
> filter is actually a python function, so you should call this variable
> something else. Also, would it be possible to take the
> Response.aggregations and find all name-bucket pairs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fabianvf/sharepa/pull/3/files#r32624148>.
>